docs/developer-docs/latest/plugins/upload.md

```diff
@@ -6,10 +6,47 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/plugins/upload.html
 
 # Upload
```

Comment:
Open question: should we mention in the TOC something like "Upload (Media Library)"? Just to make sure that when users scroll down the TOC to look for a Media Library documentation, they know that this is the right file? 🤔

Reply:
Sounds like a good idea to me :)